InnerEye/ML/model_training.py

```diff
@@ -163,6 +164,8 @@ def create_lightning_trainer(container: LightningContainer,
                       accelerator=accelerator,
                       plugins=plugins,
                       max_epochs=container.num_epochs,
+                      limit_train_batches=container.pl_limit_train_batches or 1.0,
```

Comment:
is it correct that this should be 1.0 (float)? In a few other places this param is an integer

Reply:
I've added a comment to hopefully clarify.

Reply:
Pytorch lightning seems to allow you to set this param to 0.0, in which case we'd have different behaviour from lightning.